docs/message_retention_policies.md

```diff
@@ -65,8 +65,8 @@ message retention policy configured in its state. This allows server
 admins to ensure that messages are never kept indefinitely in a server's
 database. 
 
-A default policy can be defined as such, in the `retention` section of
-the configuration file:
+A default policy can be defined as such, by adding the `retention` option in
+the configuration file and adding these sub-options:
```

Comment:
is calling sections 'options' and the options within 'sub-options' something we've already started doing?
I think I'm partial to calling them sections, but I'm not going to make a fuss of it either, it's probably just what I'm used to.

Reply:
This particular one is confusing because `retention` is both an option and a section. Essentially, there are sections which generally group functionality together (i.e. server, ratelimiting, etc), then there are options, which is the actual option/flag (ex: `modules`, and then there are sub-options, which modify a given option (ex: `module`). This is all preserved from the original documentation, I just added headers to the sections so they're easier to link to.